aspnetcore/security/authentication/jwt-authn.md

```diff
@@ -0,0 +1,155 @@
+---
+title: JWT authentication and 
+author: rick-anderson
+description: Learn how to set up manage JSON Web Tokens in development with dotnet user-jwts
+monikerRange: '>= aspnetcore-7.0'
+ms.author: riande
+ms.date: 09/22/2018
+ms.custom: "mvc, seodec18"
+uid: security/authentication/jwt
+---
+
+# Manage JSON Web Tokens in development with dotnet user-jwts
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+The `dotnet user-jwts` command line tool can create and manage app specific local [JSON Web Tokens](https://jwt.io/introduction) (JWTs).
+
+## Name
+
+`dotnet user-jwts` - Cleans the output of a project.
+
+## Synopsis
+
+```dotnetcli
+dotnet user-jwts [<PROJECT>|<SOLUTION>] [command]
+dotnet user-jwts [command] -h|--help
+```
+
+## Description
+
+Create and manages app specific local JSON Web Tokens.
```

Comment:
Here I'm missing one sentence on how the "app specific" tokens are created.
With special play-together with the JWT-middleware or how does it work.

It doesn't need a thorough description, just a one-line hint or that like.
As of now it makes me wonder how it works, so it's hard for me to put it into my "toolbox".

Reply:
I think "project-specific" might be a clearer term than app-specific. The user-jwts tool builds on some infrastructure provided by the user secrets manager, which are project specific.

So you can use the same key for the same project (which usually maps to an app) anyways.